tests/features/EditorRevisionPermissions.feature

```diff
@@ -0,0 +1,34 @@
+@api
+Feature: Editor Revision Permission Settings
+  As a user with the Editor role
+  I am able to view revision settings when adding new content if I have permission to view, revert or delete revisions
+  And if I don't have those permissions
+  I am unable to view the revision settings when adding new content
+  So I only see the settings that are available to me
+
+
+Scenario: Verifying that the Editor role has the correct permissions
+  Given I am logged in as a user with the "administrator" role
+   When I visit "/admin/people/permissions"
+    And I check the box "edit-4-view-revisions"
```

Comment:
The use of '4' is very brittle. If we ever added a new role or changed their order this would break. It would be better to go to the settings page for the administrator permissions only (as opposed to the full permissions grid) and check the boxes there.